templates/.github/rename-to-workflows/googleapis_hermetic_sync.yaml

```diff
@@ -9,7 +10,42 @@ jobs:
     - uses: actions/checkout@v3
       with:
         repository: 'googleapis/google-cloud-java'
-    - run: |
-        .kokoro/sync_with_googleapis.sh monorepo_script_output
+    - name: Setup workspace
+      run: |
+        mkdir -p "${WORKSPACE}"
+        git config user.email "yoshi-code-bot[bot]@users.noreply.github.com"
```

Comment:
Why are you calling it yoshi-code-bot?

Reply:
It's for consistency with existing job that creates pull requests https://github.com/googleapis/google-cloud-java/blob/main/.github/workflows/readme.yaml#L34

Reply:
Yoshi is a defunct term. We probably need a new convention.